chocolatey/notable.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>notable</id>
+    <version>1.3.0</version>
+    <packageSourceUrl>https://github.com/fabiospampinato/notable</packageSourceUrl>
+    <owners>Fabio Spampinato</owners>
+    <title>notable (Install)</title>
```

Comment:
Like I said, I know nothing about Chocolatey, so what you currently have might be better; but could notable be capitalised here (Notable) since it looks like it will be visible somewhere?

Reply:
Fixed!

Reply:
Yeah this was all experimental until I got it to work correctly. I had been meaning to go back and fix it.